plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -41,8 +41,21 @@
     @Option(name = "--war", aliases = {"-w"}, usage = "Path to Jenkins war file")
     private String jenkinsWarFile;
 
+    @Option(name = "--list", aliases = {"-l"}, usage = "Lists all plugins currently installed and if given a list of " +
+            "plugins to install either via file or CLI option, all plugins that will be installed by the tool",
+            handler = BooleanOptionHandler.class)
+    private boolean showPluginsToBeDownloaded;
+
+    @Option(name = "--verbose", usage = "Output more plugin dependency information",
```

Comment:
It's not just about plugin dependency versions is it? How about?
```suggestion
    @Option(name = "--verbose", usage = "Verbose logging",
```